agent/agentssh/agentssh.go

```diff
@@ -565,7 +569,12 @@ func (s *Server) CreateCommand(ctx context.Context, script string, env []string)
 	return cmd, nil
 }
 
-func (s *Server) Serve(l net.Listener) error {
+func (s *Server) Serve(l net.Listener) (retErr error) {
+	s.logger.Info(context.Background(), "started serving listener", slog.F("listen_addr", l.Addr()))
+	defer func() {
+		s.logger.Info(context.Background(), "stopped serving listener",
```

Comment:
nit: maybe we should also log the total serving time for debugging/support purposes?

Reply:
logs have timestamps, so I'm not generally worried about being able to tell how much time elapsed.

Listeners get restarted only on agent reconnections to coderd, which should be pretty obvious from the logs.

Reply:
Ok, I was thinking only about reducing the investigation time, even if the information is redundant. Feel free to leave it as is.